src/modules/fw_pos_control/FixedwingPositionControl.cpp

```diff
@@ -2102,16 +2126,7 @@ FixedwingPositionControl::control_manual_position(const float control_interval,
 		_hdg_hold_enabled = false;
 		_yaw_lock_engaged = false;
 
-		// do slew rate limiting on roll if enabled
-		float roll_sp_new = _manual_control_setpoint.roll * radians(_param_fw_r_lim.get());
-		const float roll_rate_slew_rad = radians(_param_fw_pn_r_slew_max.get());
-
-		if (control_interval > 0.f && roll_rate_slew_rad > 0.f) {
-			roll_sp_new = constrain(roll_sp_new, _att_sp.roll_body - roll_rate_slew_rad * control_interval,
-						_att_sp.roll_body + roll_rate_slew_rad * control_interval);
-		}
-
-		_att_sp.roll_body = roll_sp_new;
+		_att_sp.roll_body = _manual_control_setpoint.roll * radians(_param_fw_r_lim.get());
```

Comment:
The slew rate is now also applied in Manual Altitude mode - I don't see that as an issue, just a remark. I think generally it's better to be consistent over the whole module like you propose here.

Reply:
Yeah that is true. I also like it more consistent and that it then can properly be initialized.